packages/server-core/src/social/invite/invite.hooks.ts

```diff
@@ -45,14 +56,92 @@ import {
   inviteResolver
 } from './invite.resolvers'
 
+async function addSearch(context: HookContext<InviteService>) {
```

Comment:
Can we do what this hook does on client side? Thereby we would no longer need `search` param.

Reply:
A doubt here: I think there a lot of other services using `search` in `params.query`.
I think it is used to reduce the complexity of using the operator handling in the query.

Reply:
implemented!